stock_reorder_forecast/__openerp__.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Stock reorder forecast",
+    "version": "9.0.1.0.0",
+    "author": "Therp BV,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Stock",
+    "summary": "Predict date stock levels will reach minimum and trigger RFQ",
+    "depends": [
+        'product',
+        'stock',
+        'sale',
+        'purchase'
```

Comment:
Not blocking for me but, purchase depends of stock and sale of product, then 2 module here as dependencies is sufficient